app/services/digest_mailer.rb

```diff
@@ -0,0 +1,44 @@
+class DigestMailer
+  DIGEST_INTERVAL   = 1.day
+  INSTANT_INTERVAL  = 10.minutes
+
+  def self.send_digests
+    digest_users = User.includes(:notifications).dailies
+    digest_users.each do |user|
+      DigestMailer.new(user: user, type: :digest).send
+    end
+  end
+
+  def self.send_instants
+    instant_users = User.includes(:notifications).instants
+    instant_users.each do |user|
+      DigestMailer.new(user: user, type: :instant).send
+    end
+  end
+
+  def initialize(type:, user:)
+    @type = type
+    @user = user
+  end
+
+  attr_accessor :type, :user
+
+  def send
+    notifications = user.notifications.for_digest(interval)
+    NotificationMailer.with(user: user, notifications: notifications, type: type).
+      digest.
+      deliver_now
+  end
+
+  private
+
+  def interval
```

Comment:
I don't think this is being used.

Reply:
It's used in Line 27 for querying the notifications.